angular/demo/tsconfig.json

```diff
@@ -3,7 +3,14 @@
 	"extends": "../tsconfig.json",
```

Comment:
Should'nt all `tsconfig.json` extend from their corresponding `tsconfig.build.json` and only change the paths as it is done in angular/headless?

Reply:
thx for the comment @divdavem !
have updated the PR accordingly